main.py

```diff
@@ -151,9 +182,15 @@ def get_total_message_count(self, queues):
 
     def scale_sqs_app(self, app, paas_app):
```

Comment:
Do you want to scale the api as well?

Reply:
No, not yet at least.

Reply:
Well, on the other hand, why not? https://github.com/alphagov/notifications-paas-autoscaler/pull/26/commits/52bd2056fd3f55ce2a658e72ae0f1c7378b8e996